tardis/simulation/tests/test_simulation.py

```diff
@@ -37,6 +39,7 @@ def simulation_compare_data(simulation_compare_data_fname):
     return h5py.File(simulation_compare_data_fname, mode='r')
 
 
+@remote_data
```

Comment:
I think `@remote_data` for the `atom_data` fixture should be enough, or is there a problem?